### PR TITLE
Solid Libraries in the Solid GitHub Account

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**
re: https://github.com/solid/process/issues/169

### CODE REVIEW

**Comment by RubenVerborgh** [comment]
Suggestion: can we keep libraries and tutorials separated?
They are very different efforts, especially in terms of maintenance. Tutorials you write once; then they typically become outdated and new ones are written. (Only seldom, they are updated iteratively, since there is value in keeping the old versions around.) In contrast, creating libraries is easy compared to the effort of maintaining them.

Very different things with different needs and strategies.

File: README.md [hunk 1]
```diff
@@ -98,6 +98,16 @@ Administrators belong to the [Administrators Team](https://github.com/orgs/solid
...
 
+# Solid Libraries and Tutorials
+
+Anyone can write a Solid library and/or tutorial based on the Solid specification. 
+
+[Librarians](https://github.com/solid/process/blob/master/librarians.md) are responsible for keeping an up to date overview of the Solid Libraries and Tutorials within the ['implement' tab on solidproject.org](https://github.com/solid/solid.github.io/tree/master/pages/implement). Solid Libraries are also responsible for keeping libraries and tutorials that can be found in the Solid GitHub account up to date. 
+
+Librarians communicate about these responsibilities via [a dedicated gitter channel](https://gitter.im/solid/libraries). 
+
+[Librarians](https://github.com/solid/process/blob/master/librarians.md) are appointed by the Solid Director.
```
Comment (RubenVerborgh) [inline README.md]: I don't think this is needed at all. Anyone can create their own library anywhere in the world.

File: README.md [hunk 2]
```diff
@@ -98,6 +98,16 @@ Administrators belong to the [Administrators Team](https://github.com/orgs/solid
 
 Administrators are appointed by the Solid Director. Administrators are listed at [administrators.md](administrators.md) along with their contact details and affiliations. Anyone may apply to be an Administrator. Administrator applications are not reviewed by other Administrators. They are reviewed only by the Solid Director.
 
+# Solid Libraries and Tutorials
+
+Anyone can write a Solid library and/or tutorial based on the Solid specification. 
+
+[Librarians](https://github.com/solid/process/blob/master/librarians.md) are responsible for keeping an up to date overview of the Solid Libraries and Tutorials within the ['implement' tab on solidproject.org](https://github.com/solid/solid.github.io/tree/master/pages/implement). Solid Libraries are also responsible for keeping libraries and tutorials that can be found in the Solid GitHub account up to date.
```
Comment (RubenVerborgh) [inline README.md]: So those would not be people implementing/maintaining the libraries, but those maintaining a list of them? I would suggest for the maintenance of such a list to be a documentation task (for the sake of simplicity).

Comment (justinwb) [reply README.md]: I agree - this seems to be only documentation, so I don't see what the difference is between this and a content creator.

**Comment by jeff-zucker** [comment]
@Mitzi-Laszlo has suggested that I become a content "Creator" documenting the libraries on solidproject.org.  While that's a documentation task, I won't be able to do it without input from the implementers.  I plan to have a landing page for tools and libraries on github which, along with the gitter channel can provide a way for tool developers to collaborate with each other and provide input to the documentation process.  Does that sound like a good way to proceed?

Related questions:: Is there or should there be a distinction between tools and libraries stored in the solid repo and those stored in personal or other project repos?

**Comment by Mitzi-Laszlo** [comment]
There's nothing stopping a person applying to multiple roles. 

"The Creators are responsible for creating content for the solidproject.org website" I'll open a separate pull request clarifying Creators are focused on copy for the website.

Librarians are responsible for keeping libraries that can be found in the Solid GitHub account up to date. i.e. code maintenance.

**Pull request closed without merging**